java/core/src/main/java/co/worklytics/psoxy/impl/SanitizerImpl.java

```diff
@@ -132,6 +132,11 @@ public String sanitize(@NonNull URL url, @NonNull String jsonResponse) {
                     .delete(document, jsonConfiguration);
             }
 
+            //TODO: error handling within the map functions. any exceptions thrown within the map
+            //      function seem to be suppressed, and an empty [] left as the 'document'.
```

Comment:
Yep, it seems that  flag `Option.SUPPRESS_EXCEPTIONS` is suppressing everything, no matter where the exception is thrown.